docs/keras.rst

```diff
@@ -25,11 +25,14 @@ Horovod supports Keras and regular TensorFlow in similar ways. To use Horovod wi
 
    .. code-block:: python
 
-       gpus = tf.config.experimental.list_physical_devices('GPU')
+       device_type = 'XPU' if hvd.sycl_built() else 'GPU'
```

Comment:
From a user perspective, `sycl_built` seems very opaque. Can this method name be a bit more meaningful in the sense why am I calling it? How does `sycl` relate to the abstraction layer that provides `XPU`s?

Reply:
This example has the drawback that users that want to use `XPU`s do not realize when `hvd.sycl_built` returns false and the example silently falls back to GPU. With `device_type = 'XPU'` they will get an error in that situation and realize Horovod is not build with `sycl` support.

I'd rather prefer the example to look like
```suggestion
       device_type = 'GPU'  # use 'XPU' here if you are using Intel GPU devices with oneAPI
```

Reply:
@EnricoMi Agreed, we have changed this example docs to your suggestions. Common users only need care about "GPU" and "XPU". "sycl_built" should be transparent to users. Thanks for your review and we appreciate your any comments.

Reply:
Looks better, thanks!